bindings/pydeck/pydeck/bindings/layer.py

```diff
@@ -1,22 +1,22 @@
 import uuid
 
+import numpy as np
+
 from ..data_utils import is_pandas_df
-from .json_tools import JSONMixin
+from .json_tools import JSONMixin, camel_and_lower
 
 
-TYPE_IDENTIFIER = '@@type'
-FUNCTION_IDENTIFIER = '@@='
+TYPE_IDENTIFIER = "@@type"
+FUNCTION_IDENTIFIER = "@@="
 QUOTE_CHARS = {"'", '"', "`"}
 
 
+class BinaryTransportException(Exception):
+    pass
+
+
 class Layer(JSONMixin):
-    def __init__(
-        self,
-        type,
-        data,
-        id=None,
-        **kwargs
-    ):
+    def __init__(self, type, data, id=None, binary_transport=None, **kwargs):
```

Comment:
Is the boolean **binary_transport** needed.  Does this just come down to a **BytesIO** object (or bytes like that we can deduce from the type at runtime?)

Reply:
Right now I'm using numpy + pandas to convert to guarantee that we get a specific TypedArray on the client, so a BytesIO object may not be enough.